AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/SiteAdminController.cs

```diff
@@ -61,7 +61,9 @@ public IActionResult Index()
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> ConfirmDeleteUser(string userId)
         {
-            await _mediator.SendAsync(new DeleteUserCommand { UserId = userId });
+            var user = await _userManager.FindByIdAsync(userId);
+            await _userManager.DeleteAsync(user);

```

Comment:
@MisterJames, I didn't realize the end game was we wanted to published a `UserDeleted` event. As far as accepting this PR, I have two questions for you:

1. is there anywhere else in the system that is doing the same operation? (aka, deleting a user). If so, then we DO want this deletion code in a handler, and those two controllers that are deleting the user should send a command to do it.
2. If there are two places that delete a user, and we keep the delete call in the current handler, we should publish an event from that command handler.